### DISCUSSION TITLE
Welcome to authzapi Discussions!

### DISCUSSION THREAD

**Discussion by tulshi** [openid/authzen#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by baboulebou** [comment]
Hi there ! Alex Babeanu, CTO at 3Edges. Building and evangelizing Authorization solutions that use Graphs and Graph DBs since 2015.

**Comment by amonika230995** [comment]
Hi ! I am Monika Avalur , working at CyberArk as a product manager. Have been working on IAM since 2022.

**Comment by baboulebou** [comment]
Hi Monika, welcome to the group!

./\lex.

On Thu, Jun 26, 2025 at 7:32 AM amonika230995 ***@***.***>
wrote:

> Hi ! I am Monika Avalur , working at CyberArk as a product manager. Have
> been working on IAM since 2022.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/openid/authzen/discussions/19#discussioncomment-13586837>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABCDAWXHWFCG65FAL4D3NNL3FQABFAVCNFSM6AAAAACAGNIP2KVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTGNJYGY4DGNY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 


Alex Babeanu
Lead Product Manager, AI Control  Suite

t. +1 604 728 8130
e. ***@***.***
w. www.indykite.com
